pybaseball/team_results.py

```diff
@@ -95,8 +95,25 @@ def make_numeric(data):
 
 def schedule_and_record(season=None, team=None):
     # retrieve html from baseball reference
+    # sanatize input
+    team = team.upper()
+    df = teams()
+    seasons = df['yearID']
+    team_names = df['teamIDBR']
+    first_occurrences = team_names.drop_duplicates(keep='first')
+    first_occurrences_index = first_occurrences.index.values
+    first_years = pd.DataFrame(seasons[first_occurrences_index])
+    first_years['teamID'] = first_occurrences
+    first_season_map = dict(zip(first_years['teamID'],
+                                first_years['yearID']))
```

Comment:
Thoughts on hardcoding this instead of generating it each time the function is called? Re-downloading Lahman's teams file at each function call will be slow